azkaban-web-server/src/main/java/cloudflow/tables.sql

```diff
@@ -40,3 +40,19 @@ CREATE TABLE IF NOT EXISTS `project_admin` (
 -- ALTER TABLE execution_flows ADD COLUMN experiment_id INT DEFAULT NULL;
 -- ALTER TABLE execution_flows ADD COLUMN flow_definition_id INT;
 -- ALTER TABLE execution_flows ADD COLUMN flow_version INT;
+
+-- ***********************
+-- Required modifications of Azkaban's DB schema to support GET /flows endpoints
+-- ALTER TABLE project_flows
+-- ADD COLUMN flow_name VARCHAR(512),
+-- ADD COLUMN latest_flow_version INT;
+
+-- ALTER TABLE project_flow_files
+-- ADD COLUMN flow_id INT,
+-- ADD COLUMN experimental TINYINT(1),
+-- ADD COLUMN dsl_version TINYINT(4),
+-- ADD COLUMN created_by VARCHAR(64),
+-- ADD COLUMN locked TINYINT(1),
+-- ADD COLUMN encoding_type TINYINT(4),
+-- ADD COLUMN flow_model_blob BLOB;
+-- ***********************
```

Comment:
We should modify the CREATE TABLE syntax for the original tables as well in addition to these alter commands.

Reply:
Added the changes to create table scripts.

Reply:
I had to revert the schema changes since the build fails when the create scripts are modified. This change will have to wait till Saravanan's change is in which will ensure other queries that access the DB account for new schema.